client/jira.go

```diff
@@ -72,6 +72,14 @@ type (
 		Issues       []domain.Issue `json:"issues"'`
 	}
 
+	TransitionIdRequest struct {
+		TransitionId string `json:"id"`
```

Comment:
I use it in the `DoTransitionRequest` struct. Because the shape of the request body should look like:
```
{ 
	"transition": {
		"id": "21"
	}
}
```

Reply:
This can be simply called `Id`

Reply:
done